steveify1/nrc-hr-backend
# #146 delete employee skill

#### What does this PR do?
delete a single skill of an employee
#### Description of Task to be completed?
- implement the `deleteEmployeeSkill` controller
- write test for the endpoint
#### How should this be manually tested?
* clone the repository
* cd into the project directory
* run `npm install`
* run `npm test`
* run `npm run dev` to start the development server
* add an employee skill. see #127 #136 
* send a `DELETE` request to `/employee/:ippisNo/skill/:employeeSkillId`
#### Any background context you want to provide?
* this request requires authentication and authorization; a `token` header containing the client's token must be sent along with it. See #38 and #39 for how to acquire a token.
#### What are the relevant pivotal tracker stories?
#146 
#### Screenshots (if appropriate)
#### Questions:
